### PR TITLE
Issue/6 7/login logout menu items

### PR DESCRIPTION
**Pull request by blohinn**
Implemented base urls to login/logout in flask admin menu. 
add base template for future extensibility.

issues:
#6 
#7

### CODE REVIEW

File: mg_incident/__init__.py [hunk 1]
```diff
@@ -16,6 +17,9 @@
 from mg_incident.ticket import admin_views as ticket_admin_views
 from mg_incident.feedback import admin_views
 
+admin.add_link(MenuLink(name='Google', category='Links', url='http://www.google.com/'))
```
Comment (vstpnv) [inline mg_incident/__init__.py]: Why is it here?

File: mg_incident/__init__.py [hunk 2]
```diff
@@ -16,6 +17,9 @@
...
+admin.add_link(MenuLink(name='Mozilla', category='Links', url='http://mozilla.org/'))
```
[the remark this answers is not in the document]
Comment (blohinn) [reply mg_incident/__init__.py]: oops, updated

File: mg_incident/templates/base.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+{% block access_control %}
+{% if current_user.is_authenticated %}
+<div class="navbar-text btn-group pull-right">
+    <a href="{{ url_for('security.logout') }}">Выйти</a>
+</div>
+{% endif %}
+<div class="navbar-text btn-group pull-right">
+    <a href="{{ url_for('security.login') }}">Войти</a>
```
Comment (vstpnv) [inline mg_incident/templates/base.html]: Is it visible to authenticated users? Why?

File: mg_incident/templates/base.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+{% endblock head_css %}
+
+{% block head %}
+{% endblock head %}
+
+
+{% block access_control %}
+{% if current_user.is_authenticated %}
+<div class="navbar-text btn-group pull-right">
+    <a href="{{ url_for('security.logout') }}">Выйти</a>
```
Comment (vstpnv) [inline mg_incident/templates/base.html]: English, please

**Pull request merged**